src/lib/Components/Browse.php

```diff
@@ -32,26 +32,13 @@ protected function getLocationId()
         return "false";
     }
 
-    protected function getLocationRestId()
-    {
-        if ($this->request->attributes->get('location')) {
-            return $this->urlGenerator->generate(
```

Comment:
together with that change, you should also remove the inline comment above the constructor